import/repos.json

```diff
@@ -22,7 +22,7 @@
       },
       {
         "version": "v5",
-        "name": "release/oss-v5"
+        "name": "fixes_another_dead_link_in_v5_docs"
```

Comment:
This should be reverted before the merge, once https://github.com/EventStore/EventStore/pull/3045 is merged.